src/App.jsx

```diff
@@ -1,8 +1,123 @@
 import "./App.css";
 import Todo from "./components/Todo";
+import { useEffect, useState } from "react";
+import { nanoid } from "nanoid";
 
 function App() {
-    return <div className="main bg-lime-400"></div>;
+    const [todos, setTodos] = useState(() => {
+        return window.localStorage.getItem("todos")
+            ? [...JSON.parse(window.localStorage.getItem("todos"))]
+            : [{ text: "initial todo", id: null, checkness: false }];
+    });
+    const [currentTodo, setCurrentTodo] = useState({ text: "", id: null });
+    const [editMode, setEditMode] = useState(false);
+    const increaseTodos = () => {
+        if (currentTodo.text.trim() === "") return;
+        setTodos([
+            ...todos,
+            { text: currentTodo.text.trim(), id: nanoid(), checkness: false },
+        ]);
+        setCurrentTodo({ text: "", id: null });
+    };
+    const deleteOneTodo = (id) => {
+        let indexToDelete = todos.findIndex((el) => el.id === id);
```

Comment:
Use descriptive name, it can be hard for other developers to know what you mean by "el".